utils/utils.go

```diff
@@ -2,40 +2,47 @@ package utils
 
 import (
 	"context"
-    "os"
-    "time"
-    "log"
-
-    "cod/debug"
-
-    v1 "k8s.io/api/core/v1"
-    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-    "k8s.io/apimachinery/pkg/runtime/schema"
-    "k8s.io/client-go/dynamic"
-    "k8s.io/client-go/dynamic/dynamicinformer"
-    "k8s.io/client-go/kubernetes"
-    "k8s.io/client-go/tools/cache"
-
+	"log"
+	"os"
+	"time"
+
+	"cod/debug"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
 )
 
+type Message struct {
+	Type        string `json:"type"`
+	ClusterName string `json:"clusterName"`
+	Name        string `json:"name"`
+	Message     string `json:"message"`
+	Kind        string `json:"kind"`
+	ObjectName  string `json:"objectName"`
+	SessionID   string `json:"sessionId,omitempty"`
+}
 
 // getPodLabels retrieves labels of an involved pod
 func GetPodLabels(clientset *kubernetes.Clientset, dynamicClient dynamic.Interface, obj v1.ObjectReference) map[string]string {
-    if obj.Kind == "Pod" {
-        debug.Println("Fetching labels for pod:", obj.Name)
-        pod, err := clientset.CoreV1().Pods(obj.Namespace).Get(context.TODO(), obj.Name, metav1.GetOptions{})
-        if err != nil {
-            debug.Println("Error fetching pod:", err)
-            return nil
-        }
-        return pod.Labels
-    }else{
-        debug.Println("Unsupported kind:", obj.Kind)
-        return nil
-    }
+	if obj.Kind == "Pod" {
+		debug.Println("Fetching labels for pod:", obj.Name)
+		pod, err := clientset.CoreV1().Pods(obj.Namespace).Get(context.TODO(), obj.Name, metav1.GetOptions{})
+		if err != nil {
+			debug.Println("Error fetching pod:", err)
+			return nil
+		}
+		return pod.Labels
+	} else {
+		debug.Println("Unsupported kind:", obj.Kind)
+		return nil
+	}
 }
 
-
 // func GetObjectLabels(clientset *kubernetes.Clientset, dynamicClient dynamic.Interface, obj v1.ObjectReference) map[string]string {
```

Comment:
large segments of commented code generally should be removed from the code base, unless the comment serves a purpose to explain other code, or give an example of what to do or not to do.  Otherwise it can be difficult to use searching tools to find specific areas of code, if there are repeated sections that match that are all commented out.